packages/@lwc/integration-karma/test/accessibility/synthetic-cross-root-aria/index.spec.js

```diff
@@ -37,7 +37,15 @@ if (!process.env.NATIVE_SHADOW) {
                 targetElm = elm.shadowRoot.querySelector('x-aria-target');
             });
 
-            [false, true].forEach((usePropertyAccess) => {
+            const usePropertyAccessValues = [false];
+
+            // It doesn't make sense to test setting e.g. `elm.ariaLabelledBy` if the global
+            // polyfill is not applied
+            if (window.lwcRuntimeFlags.ENABLE_ARIA_REFLECTION_GLOBAL_POLYFILL) {
+                usePropertyAccessValues.push(true);
+            }
```

Comment:
This test was busted before, but we didn't notice because we weren't testing global ARIA reflection in native shadow mode.